public/index.php

```diff
@@ -7,50 +7,110 @@
  * @author     Fuel Development Team
  * @license    MIT License
  * @copyright  2010 - 2018 Fuel Development Team
- * @link       http://fuelphp.com
+ * @link       https://fuelphp.com
  */
 
 /**
- * Set error reporting and display errors settings.  You will want to change these when in production.
+ * -----------------------------------------------------------------------------
+ *  Error Reporting
+ * -----------------------------------------------------------------------------
+ *
+ *  Display error details when something is wrong.
+ *
+ *  It is recommended to disable error reporting when in production. To disable
```

Comment:
This recommendation is not correct. 

Disabling this only causes the error handler not being able to log issues. Fuel doesn't use these values the PHP way, everything this handled in the ErrorHandler. 

The recommendation should be not to change them.

Reply:
Ah, yes, thank you.  

So, I just have to remove these code from docbloc?

```
error_reporting(0);
ini_set('display_errors', 0);
```